src/views/Swap/index.tsx

```diff
@@ -12,13 +12,15 @@ import {
   BottomDrawer,
   useMatchBreakpoints,
   ArrowUpDownIcon,
+  Skeleton,
 } from '@pancakeswap/uikit'
 import { useIsTransactionUnsupported } from 'hooks/Trades'
 import UnsupportedCurrencyFooter from 'components/UnsupportedCurrencyFooter'
 import Footer from 'components/Menu/Footer'
 import { useRouter } from 'next/router'
 import { useTranslation } from 'contexts/Localization'
 import SwapWarningTokens from 'config/constants/swapWarningTokens'
+import { useFetchBlockNumber } from 'state/block/hooks'
```

Comment:
this hook with delay loading is only for swap page only, and `useFetchBlockNumber` naming is quite misleading. 
I recommend move it to swap views folder. state related code is not easy to tree shaking base on pages right now.